app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -260,19 +355,22 @@ class Exposure extends React.Component {
           </Form.Group>
         </Form.Row>
         <Form.Row>
-          <Form.Group>
+          <Form.Group className="ml-1">
             <Form.Check
               size="lg"
-              label="CONTINUOUS EXPOSURE"
-              type="switch"
+              label={`CONTINUOUS EXPOSURE${schema?.fields?.continuous_exposure?._whitelist?.list?.has(true) ? ' *' : ''}`}
               id="continuous_exposure"
-              className="ml-1"
-              checked={this.state.current.patient.continuous_exposure}
+              className="ml-1 d-inline"
+              checked={this.state.current.patient.continuous_exposure || false}
```

Comment:
I think it was because I was getting one of these errors: https://stackoverflow.com/questions/47012169/a-component-is-changing-an-uncontrolled-input-of-type-text-to-be-controlled-erro

but I will change it to what you suggested, thanks!